.github/.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{  
+  "files.autoSave": "afterDelay",
+  "files.autoSaveDelay": 1000,  
+}
```

Comment:
I'm not sure this is something we should turn on for everyone who wants to work in the repo. While I think it may be helpful for some, I'm not sure I see this as intuitive or expected behavior. If there is something I don't want to save, I need to remember to explicitly *un-save* it somehow or revert the behavior.

This seems more like something that should be added in user settings rather than as default behavior for the repo.

Reply:
This issue was created a long time ago. I didn't think It was necessary. But, can we guarantee that everyone will do the settings always for autosave?. As we always forget to run yarn or commit before pushing to master. 
@John Kreitlow created the issue and he has suggested some workspace config settings. but I only find the auto save necessary.  Let see what he thinks about the PR.  if not necessary still, we can close it.

Reply:
I added a [comment to the issue](https://github.com/microsoft/fast/issues/3136#issuecomment-816007158) with some suggestions for defaults. We don't want to set anything that could interfere with a developer's regular way of working, we only want to enforce some formatting-specific settings to reduce issues when working in the project.

Reply:
@radium-v given you originally created this issue, I think, would you mind specifying the defaults you were thinking that makes sense and we can just do those for now?  Thanks

Reply:
The link in my previous comment points to a set of suggested defaults I added to the issue. While there shouldn't be any harm in adding them, I would like to get more input from the team before committing anything.